### PR TITLE
tests: add integration tests for EE

### PR DESCRIPTION
**Pull request by GGabriele**

### CODE REVIEW

File: .github/workflows/integration-enterprise.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    env:
+      KONG_ANONYMOUS_REPORTS: "off"
+      KONG_IMAGE: ${{ matrix.kong_image }}
+      KONG_LICENSE_DATA: ${{ secrets.KONG_LICENSE_DATA }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup go
+        uses: actions/setup-go@v2
+        with:
+          go-version: '^1.16'
```
Comment (hbagdi) [inline .github/workflows/integration-enterprise.yaml:27]:
```suggestion
          go-version: '^1.17'
```

File: .ci/setup_kong_ee.sh [hunk 2]
```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+set -e
+
+export KONG_LICENSE_PATH=/tmp/license.json
```
Comment (hbagdi) [inline .ci/setup_kong_ee.sh:5]: Unused

File: .ci/setup_kong_ee.sh [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  -p 8443:8443 \
+  -p 8001:8001 \
+  -p 8444:8444 \
+  -p 8002:8002 \
+  -p 8445:8445 \
+  -p 8003:8003 \
+  -p 8004:8004 \
+  $KONG_IMAGE
+
+sleep 10
```
Comment (hbagdi) [inline .ci/setup_kong_ee.sh:66]: Replace this and the other sleep with some netcat retry loops please. Do this for the OSS setup as well.

File: .ci/setup_kong_ee.sh (hunk 2 above)
Comment (GGabriele) [reply .ci/setup_kong_ee.sh]: Ops, leftover

File: .ci/setup_kong_ee.sh (hunk 3 above)
Comment (GGabriele) [reply .ci/setup_kong_ee.sh]: done

File: .github/workflows/integration-enterprise.yaml (hunk 1 above)
Comment (GGabriele) [reply .github/workflows/integration-enterprise.yaml]: updated this on other CI too

File: .ci/setup_kong.sh [hunk 4]
```diff
@@ -25,7 +33,7 @@ docker run --rm -d --name $PG_CONTAINER_NAME \
   -e "POSTGRES_PASSWORD=$KONG_DB_PASSWORD" \
   postgres:9.6
 
-sleep 5
+waitContainer "PostgreSQL" 8001 5
```
Comment (hbagdi) [inline .ci/setup_kong.sh:36]: poll aggressively everywhere, something like 0.2

Comment (GGabriele) [reply .ci/setup_kong.sh]: done

**Review by hbagdi — APPROVED** [review]

**Pull request merged**